adapter/src/main/kotlin/de/fraunhofer/iem/spha/adapter/kpis/cve/CveAdapter.kt

```diff
@@ -17,14 +17,26 @@ import de.fraunhofer.iem.spha.model.kpi.RawValueKpi
 
 object CveAdapter {
 
-    fun transformDataToKpi(data: Collection<VulnerabilityDto>): Collection<AdapterResult> {
+    fun transformCodeVulnerabilityToKpi(
+        data: Collection<VulnerabilityDto>
+    ): Collection<AdapterResult> {
+        return transformDataToKpi(data, KpiId.CODE_VULNERABILITY_SCORE)
+    }
+
+    fun transformContainerVulnerabilityToKpi(
+        data: Collection<VulnerabilityDto>
+    ): Collection<AdapterResult> {
+        return transformDataToKpi(data, KpiId.CONTAINER_VULNERABILITY_SCORE)
+    }
+
+    private fun transformDataToKpi(
+        data: Collection<VulnerabilityDto>,
+        kpiId: KpiId,
+    ): Collection<AdapterResult> {
         return data.map {
             return@map if (isValid(it)) {
                 AdapterResult.Success.Kpi(
-                    RawValueKpi(
-                        kind = KpiId.VULNERABILITY_SCORE,
-                        score = (it.severity * 10).toInt(),
-                    )
+                    RawValueKpi(kind = kpiId, score = (it.severity * 10).toInt())
```

Comment:
I think this is missing the score fix from the other PR.
```suggestion
                    RawValueKpi(kind = kpiId, score = 100 - (it.severity * 10).toInt())
```

Reply:
yep. The fix will be integrated once I merge main back into this branch. I'm just waiting with this until we finish up the other review and merge.